actors/miner/tests/precommit_deposit_and_initial_pledge_positive_test.rs

```diff
@@ -0,0 +1,45 @@
+use fil_actor_miner::{initial_pledge_for_power, pre_commit_deposit_for_power};
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::sector::StoragePower;
+use fvm_shared::smooth::FilterEstimate;
+use num_traits::zero;
+
+macro_rules! my_const {
```

Comment:
This is really really nice.  I'd love to go through and apply this all over the other tests where rust has stopped us from defining constants of types with destructors.

Reply:
Sure, I can look into it in another PR. :)